eden/scm/edenscm/ext/github/pull_request_body.py

```diff
@@ -55,10 +56,16 @@ def create_pull_request_title_and_body(
         ]
     )
     title = firstline(commit_msg)
-    body = f"""{commit_msg}
+    if reviewstack:
+        body = f"""{commit_msg}
 {_HORIZONTAL_RULE}
 Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack]({reviewstack_url}).
 {bulleted_list}
+"""
+    else:
+        body = f"""{commit_msg}
+{_HORIZONTAL_RULE}
+{bulleted_list}
```

Comment:
I think we can skip the list and line for single-commit stack. So I updated the code to do that.